src/main/scala/encry/local/miner/EncryMiningWorker.scala

```diff
@@ -32,7 +32,7 @@ class EncryMiningWorker(myIdx: Int, numberOfWorkers: Int) extends Actor with Log
     case NextChallenge(candidate: CandidateBlock) =>
       challengeStartTime = new Date(System.currentTimeMillis())
       context.become(miningInProgress)
-      log.info(s"Start challenge on worker: $myIdx on height ${candidate.parentOpt.map(_.height + 1).getOrElse("No parent")} in ${sdf.format(challengeStartTime)}")
+      log.info(s"Start challenge on worker: $myIdx at height ${candidate.parentOpt.map(_.height + 1).getOrElse("No parent")} at ${sdf.format(challengeStartTime)}")
```

Comment:
Maybe change it so the log won't a contain string with "at height No parent" in case that parentOpt is empty?